integration/tools/validation/src/main/java/alluxio/cli/UnderFileSystemContractTest.java

```diff
@@ -57,6 +59,10 @@
       description = "The under filesystem path to run tests against.")
   private String mUfsPath;
 
+  @Parameter(names = {"--test"}, required = false,
+      description = "Test name, use this option multiply time to run multiply test")
```

Comment:
`use this option multiply time` what does this mean?

Reply:
It mean that, we can use it something like this,
```
alluxio runUfsTests --test test1 --test test2 --test test3
```

Reply:
multiply -> multiple